libs/oidc/common/src/lib/services/user/user.service.ts

```diff
@@ -30,8 +30,8 @@ import * as deepmerge from 'deepmerge';
 
 @Injectable()
 export class UserService {
-  private IPSTACK_APIKEY = '1e599a1240ca8f99f0b0d81a08324dbb';
-  private IPSTACK_URL = 'http://api.ipstack.com/';
+  // private IPSTACK_APIKEY = '1e599a1240ca8f99f0b0d81a08324dbb';
+  // private IPSTACK_URL = 'http://api.ipstack.com/';
```

Comment:
No secret keys of any kind in version control

Reply:
The EnvironmentService would work great here, but seems to be limited to Angular projects only. Is there an existing EnvironmentService for Nest or should I consider making one?

Reply:
There is not one for Nest. I can't remember correctly but I *think* I tried implementing one some time ago and had an issue with injection tokens not being available outside the module where they were set which put a damper on being able to declare the environment on the root module and accessing it on some child module downstream. Not sure if anything's changed. 

But yes, either a service needs to be made OR just pass those values down as provider tokens. I don't think this PR should be dependent on an additional project so for now I say just go with the latter and we can figure out the environment service later. There's an example in the `veoride-data-api.module.ts` file.